Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt-get update -qq && \
     apt-get install -yqq git cmake doxygen yasm nasm pkg-config libx11-dev python3 python3-pip libasound2-dev libv4l-dev && \
     pip3 install six pystache && \
     ln -s /usr/bin/python3 /usr/bin/python && \
-    git clone --depth 1 --branch release/$LINPHONE_VERSION https://gitlab.linphone.org/BC/public/linphone-sdk.git linphone-sdk && \
+    git clone --recurse-submodules --branch release/$LINPHONE_VERSION https://gitlab.linphone.org/BC/public/linphone-sdk.git linphone-sdk && \
```

Comment:
Any reason to remove the `--depth 1`?
If we need submodules we should also add `--shallow-submodules`
Their git history is really huge and painful to clone

Reply:
@sopelj

Reply:
Sorry, it was a while ago. I seem to recall it would not clone the submodules correctly with it, but I'll try again. We do need submodules or it fails, `--shallow-submodules` is a good idea, I did not know it existed.

Reply:
Ok, so, with `--depth 1` it fails to build because they use tags and the history in the building process. I added `--single-branch` and `--shallow-submodules` though.